rosidl_generator_py/cmake/find_valid_typesupport.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
Most CMake files have one function / macro per file and their names match: https://github.com/ament/ament_cmake/tree/master/ament_cmake_core/cmake/environment

They all have a namespace like prefix depending on the context it might not be the full package name. E.g. MAny ament functions just start with `ament_`: e.g. https://github.com/ament/ament_cmake/blob/master/ament_cmake_export_libraries/cmake/ament_export_library_names.cmake

For this case maybe `rosidl_generator_py_get_typesupports(...)` (since I don't see a good way to shorten the package name)?